cli/command/trust/info.go

```diff
@@ -119,23 +119,27 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	}
 
 	// This will always have the root and targets information
-	fmt.Fprintf(cli.Out(), "\nList of admins and their KeyIDs:\n\n")
-	printSignerInfo(cli, adminRoleToKeyIDs)
+	fmt.Fprintf(cli.Out(), "\nRepository keys for %s:\n", repoInfo.Name)
+	for name, key := range adminRoleToKeyIDs {
+		fmt.Fprintf(cli.Out(), "%s:\t%s\n", name, key)
+	}
 
 	return nil
 }
 
 // Extract signer keys and admin keys from the list of roles
-func getSignerAndAdminRolesWithKeyIDs(roleWithSigs []client.RoleWithSignatures) (map[string][]string, map[string][]string) {
+func getSignerAndAdminRolesWithKeyIDs(roleWithSigs []client.RoleWithSignatures) (map[string][]string, map[string]string) {
 	signerRoleToKeyIDs := make(map[string][]string)
-	adminRoleToKeyIDs := make(map[string][]string)
+	adminRoleToKeyIDs := make(map[string]string)
 
 	for _, roleWithSig := range roleWithSigs {
 		switch roleWithSig.Name {
 		case trust.ReleasesRole, data.CanonicalSnapshotRole, data.CanonicalTimestampRole:
 			continue
-		case data.CanonicalRootRole, data.CanonicalTargetsRole:
-			adminRoleToKeyIDs[notaryRoleToSigner(roleWithSig.Name)] = roleWithSig.KeyIDs
+		case data.CanonicalRootRole:
+			adminRoleToKeyIDs["Signer Admin Key"] = roleWithSig.KeyIDs[0]
+		case data.CanonicalTargetsRole:
+			adminRoleToKeyIDs["Root Pinning Key"] = roleWithSig.KeyIDs[0]
```

Comment:
we can technically have multiple root keys, so perhaps we could do a join with `", "` separator here?  I don't anticipate it will be the common case but we should cover it.

It's also possible for targets though not exposed, so perhaps we can defensively add the join there as well?